openedx/features/assessment/tests/test_helpers.py

```diff
@@ -116,7 +121,7 @@ def test_find_and_autoscore_submissions_successfully(self, mock_get_value, mock_
         submission_uuids = uuid_list_1 + uuid_list_2
         delta_datetime = datetime_now - timedelta(days=3)
```

Comment:
e.g days_to_wait for 3

Reply:
`timedelta` is not a custom function it is a Django `datetime` function

Reply:
[timedelta](https://docs.python.org/3/library/datetime.html#timedelta-objects) is not a custom function it is a Django `datetime` function